src/hooks/useCart/useCart.test.tsx

```diff
@@ -596,6 +600,95 @@ describe("useCart", () => {
       ]);
     });
 
+    it("should set error with message 'Please enter unique codes to checkout' when identifier values are identical", async () => {
```

Comment:
haha do you wanna add a suffix, so this becomes 

`
should set error with message 'Please enter unique codes to checkout' when identifier values are identical in the same category
`

Reply:
okay! didnt change to after i added the different category test case oops thanks!!